props/saber.h

```diff
@@ -21,7 +21,11 @@ Start Soundtrack -        Long-click the POW button while blade is off.
 Blaster block -           Short-click AUX button.
 Enter/Exit Color Change - 1 button saber = Hold button and Twist.
                           2 button saber = Hold Aux and click POW while on.
-** Note Color Change only works with ProffieOS 3.x and above)
+** Note Color Change only works with ProffieOS 3.x and above
+Enter Menu -              Double-click AUX button while blade is off.
+  Use POW to select
+  Use AUX to cancel/exit
+** Note Menu only works with ProffieOS 8.x and above and requires a voicepack V2 in your common folder
```

Comment:
You don't need to mention ProffieOS 8.x, because this file is *in* ProffieOS 8.x.